gulpfile.js

```diff
@@ -122,26 +122,33 @@ gulp.task('scripts', [ 'eslint' ], function () {
 
 gulp.task('images', function () {
   gutil.log(gutil.colors.cyan('images'), 'Copying image assets');
-  return gulp.src('./assets/images/**/*')
+  return gulp.src([
+      './assets/images/**/*',
+      './node_modules/uswds/src/img/**/*'
+    ])
     .pipe(gulp.dest('./static/assets/images'));
 });
 
 gulp.task('fonts', function () {
   gutil.log(gutil.colors.cyan('fonts'), 'Copying font assets');
-  return gulp.src('./assets/fonts/**/*')
+  return gulp.src([
+      './assets/fonts/**/*',
+      './node_modules/uswds/src/fonts/**/*'
+    ])
     .pipe(gulp.dest('./static/assets/fonts'));
 });
 
 gulp.task('build', [ 'clean-all' ], function (done) {
   printPackageInfo();
   gutil.log(gutil.colors.cyan('build'), 'Building asset-pipeline');
-  runSequence([ 'styles:homepage', 'scripts' ], done);
+  runSequence([ 'styles:homepage', 'scripts', 'images' ], done);
```

Comment:
We should add `fonts` to this array again so that it can happen on build.

```
diff --git a/gulpfile.js b/gulpfile.js
index 00f4c6c..bec4f92 100644
--- a/gulpfile.js
+++ b/gulpfile.js
@@ -141,7 +141,7 @@ gulp.task('fonts', function () {
 gulp.task('build', [ 'clean-all' ], function (done) {
   printPackageInfo();
   gutil.log(gutil.colors.cyan('build'), 'Building asset-pipeline');
-  runSequence([ 'styles:homepage', 'scripts', 'images' ], done);
+  runSequence([ 'styles:homepage', 'scripts', 'images', 'fonts' ], done);
 });

 gulp.task('watch', function () {
```